### ISSUE TITLE
Quiet mode - hide repos that are already up to date

### ISSUE THREAD

**Issue by elyscape** [earwig/git-repo-updater#44]
I have several hundred repos in my gitup bookmarks, and many of them don't have updates when I run `gitup`. It would be nice if there were a way to have `gitup` stifle output for repos that didn't have any updates.

**Comment by earwig** [comment]
Yes, this is a good idea: I did some work on it in response to #38 but got distracted and never finished. I will get around to it eventually, I promise!

**Comment by PilleOnline** [comment]
Any chance on this being done? I have a lot of repos checked out, and it's enough for me to see the the errors only (and maybe a statistic at the end (6 updated, 3 error, 39 without updates, ...))

**Comment by bendog** [comment]
@PilleOnline are you still chasing this? 

I had a look through the source code today and I suspect it shouldn't be too hard to try and mute some print statements

**Comment by PilleOnline** [comment]
no, I've written a dirty bash script that fulfills my needs, and have completely forgetten about this ...
